src/testing/byzantine.rs

```diff
@@ -52,7 +52,7 @@ impl<'a> MaliciousMember<'a> {
         }
     }
 
-    fn unit_to_data(su: SignedUnit<'a, Hasher64, Data, KeyBox>) -> NetworkData {
+    fn unit_to_data(su: SignedUnit<Hasher64, Data, KeyBox>) -> NetworkData {
```

Comment:
Uh, what? How does this not have `'a` now? Does this even compile? If not then I'm kinda worried that our CI does not catch it. <_<

Reply:
[Lifetime ellision](https://doc.rust-lang.org/reference/lifetime-elision.html). Although it does compile, `SignedUnit<'_, Hasher64, Data, KeyBox>` is preferred according to rust documentation